### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by johancvb**
- Edited description to more 'gender-neutral' description
- Added the install-manual
- Added license

Feel free to give some feedback on grammar :)

P.S.
Love the LanguageTool, @SharonV33 😋

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -156,8 +154,62 @@ Here is a screenshot of the map-structure in our project:<br><br>
...
 https://bradfrost.com/blog/post/atomic-web-design/
 
 ## © License
+
+This repo is licensed as MIT by Veerle Prins, Sharon Veldman and Johan van Beusekom, 2021.
+<br><br><br>
+
+MIT License
+
+Permission is hereby granted, free of charge, to any person obtaining a copy
```
Comment (SharonV33) [inline README.md]: I believe you don't have to include the full license, this license file is also in the repo. You only have to mention that the project is under mit license and our names

**Review by SharonV33 — COMMENTED** [review]
Looking good, but please check the license part in the readme.

File: README.md (hunk 1 above)
Comment (johancvb) [reply README.md]: Thanks :)
Removed it!

File: README.md [hunk 2]
```diff
@@ -13,21 +13,21 @@ _by [Sharon Veldman](https://github.com/SharonV33), [Johan van Beusekom](https:/
...
 
-That's where FemAlert comes in handy. FemAlert is an initiative, established by Rosanne Verbeek, to protect people (mostly women) from unwanted situations while partying or going out to clubs.
+That's where FemAlert comes in handy. FemAlert is an initiative, established by Rosanne Verbeek, to protect people from unwanted situations while partying or going out to clubs.
 FemAlert is a platform where people can report any kind of unacceptable behavior in an accessible way, without immediately taking it to the police. FemAlert also lets club or bar owners get in touch with victims to help them and offer support.
 
 The platform FemAlert is also made for club owners to give them insights, to see exactly what's going on inside of their clubs. With these insights, club owners can anticipate on the situations to make the nightlife much safer and better in their venues.
 
-Our job is to make a neat-designed and detailled dashboard for club owners to provide them with visualized data about the reports made in/for their clubs.
+Our job is to make a neat-designed and detailed dashboard for club owners to provide them with visualized data about the reports made in/for their clubs.
```
Comment (veerleprins) [inline README.md]: Maybe you can put here 'for both club owners and festival owners'

File: README.md [hunk 3]
```diff
@@ -107,46 +107,44 @@ These visualizations provide the most important data for club owners, and give t
...
 
-As you can see on the screenshot of our design, we have implemented a searchbar to find specific reports. With the search bar, you can find reports for specific days and times as well as a specific type of harassment.
+As you can see on the screenshot of our design, we have implemented a search bar to find specific reports. With the search bar, you can find reports for specific days and times, as well as a specific type of harassment.
 
 Next to the section with all the reports, you can see two pie charts. These charts visualize the amount of victims who asked for help or want to be in contact with FemAlert.
 
 ### 🔩 Settings
 
-On this page, the settings of the club owner are shown. The club owners can edit their information here, or turn on the Dark theme mode. Here's a screenshot of our design for the 'Settings' page:<br><br>
+On this page, the settings of the club owner are shown. The club owners can edit their information here, log out, or turn on the Dark theme mode. Here's a screenshot of our design for the 'Settings' page:<br><br>
```
Comment (veerleprins) [inline README.md]: Maybe you can put here 'club owner or festival owner'

File: README.md [hunk 4]
```diff
@@ -156,8 +154,41 @@ Here is a screenshot of the map-structure in our project:<br><br>
...
+`npm run dev`
+<br>
+
 ## 📋 Sources
 
 https://bradfrost.com/blog/post/atomic-web-design/
 
 ## © License
+
+This repo is licensed as MIT by Veerle Prins, Sharon Veldman and Johan van Beusekom, 2021.
```
Comment (veerleprins) [inline README.md]: Maybe you can make links from this (and link to our GitHub pages) 😄

File: README.md [hunk 5]
```diff
@@ -156,8 +154,41 @@ Here is a screenshot of the map-structure in our project:<br><br>
...
+To start off, install Node.js & npm. Node.js is a platform built on Chrome's JavaScript runtime for easily building fast and scalable network applications. Npm is a Node Package Manager, to install and manage packages.
+
+To install Node.js and npm, go to <a href="https://nodejs.org/en/">https://nodejs.org/en/</a>, and download the latest version of Node.js on your device. Make sure to select the npm-install option while installing Node.js.
+
+### 📍 Clone the GitHub repository
+
+To get the project on your code-editor, you have to clone it from our GitHub repository. (<a href="https://github.com/veerleprins/FemAlert-meesterproef">https://github.com/veerleprins/FemAlert-meesterproef</a>)
+<br>
+To do this, type the following command in your terminal:
+<br><br>
```
Comment (veerleprins) [inline README.md]: All <br> 's can be removed. Instead, you can put 4 spaces on the line above and enter twice! Then you get the same effect 😄

File: README.md [hunk 6]
```diff
@@ -107,46 +107,44 @@ These visualizations provide the most important data for club owners, and give t
...
-D3.js is a JavaScript libary to visualize data and make various types of graphs, charts, and other visualizations.
+D3.js is a JavaScript library to visualize data and make various types of graphs, charts, and other visualizations.
 <br>
 
 Our working method in this project is based on an article written by Brad Frost. In this article, Brad Frost writes about Atomic Web Design. Atomic Web Design is a work method where you basically filter your application in multiple layers. <br><br>
-With Atomic Web Design, you are using Atoms, Molucules, Organisms, and Templates. Each group is a part of the next group. For example; multiple atoms form a molecule, multiple molecules form an organism, and so on. On the following picture you can see a visual representation of Atomic Web Design: <br>
+With Atomic Web Design, you are using Atoms, Molecules, Organisms, and Templates. Each group is a part of the next group. For example; multiple atoms form a molecule, multiple molecules form an organism, and so on. On the following picture, you can see a visual representation of Atomic Web Design: <br>
 
 <img src="https://imgur.com/zJcPlv4.png" width="500" height="300">
```
Comment (veerleprins) [inline README.md]: I think this is the image of the atomic web design right? That image can be removed! Instead just briefly explain what it is 😸

**Review by veerleprins — CHANGES_REQUESTED** [review]
Would you like to look at the short feedback points and be able to adjust them? Other than that it looks good! 👍🏼

File: README.md [hunk 7]
```diff
@@ -139,12 +139,9 @@ Svelte is a JavaScript framework made to reduce boilerplate code. This makes you
 D3.js is a JavaScript library to visualize data and make various types of graphs, charts, and other visualizations.
 <br>
```
Comment (veerleprins) [inline README.md]: This < br > can also be removed

File: README.md [hunk 8]
```diff
@@ -139,12 +139,9 @@ Svelte is a JavaScript framework made to reduce boilerplate code. This makes you
...
 
-Our working method in this project is based on an article written by Brad Frost. In this article, Brad Frost writes about Atomic Web Design. Atomic Web Design is a work method where you basically filter your application in multiple layers. <br><br>
-With Atomic Web Design, you are using Atoms, Molecules, Organisms, and Templates. Each group is a part of the next group. For example; multiple atoms form a molecule, multiple molecules form an organism, and so on. On the following picture, you can see a visual representation of Atomic Web Design: <br>
+Our working method in this project is based on an article written by Brad Frost. In this article, Brad Frost writes about Atomic Web Design. Atomic Web Design is a work method where you basically filter your application in multiple layers. <a href="https://bradfrost.com/blog/post/atomic-web-design/">Click here for the article.</a><br>
```
Comment (veerleprins) [inline README.md]: This < br > can also be removed!

**Review by veerleprins — APPROVED** [review]
Looks good 👍🏼  Good work!

**Review by SharonV33 — APPROVED** [review]
Nice work!

**Pull request merged**